src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -2039,6 +2068,9 @@
   - name: Reimage VMs within a VMSS
     text: az vmss reimage --instance-id instanceId --name MyScaleSet --no-wait --resource-group MyResourceGroup
     crafted: true
+  - name: Reimage VMs within a VMSS. (autogenerated)
+    text: az vmss reimage --instance-id instanceId --name MyScaleSet --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
Instance ID should just be an integer, so this value is not correct.

Reply:
I made a commit to fix this, since we don't want to create another PR. I've also made a change to the pipeline to resolve this issue permanently.